extension/data_loader/file_data_loader.h

```diff
@@ -26,6 +26,24 @@ namespace extension {
  */
 class FileDataLoader final : public executorch::runtime::DataLoader {
  public:
+  /**
+   * Creates a new FileDataLoader that wraps the named file descriptor.
+   *
```

Comment:
Should be mention that the fd ownership will be passed?

Reply:
does passing the FD ownership mean that ET is responsible for closing the file afterwards? or is ownership of the FD independent of ownership of the file opened?